assets/javascripts/discourse/templates/components/translate-topic-title.hbs

```diff
@@ -0,0 +1,4 @@
+<a {{action "translate"}} title={{i18n "translator.view_translation"}} class="translate-title" role="button">
```

Comment:
> I am seeing this button even when my interface language is English (the post translation button is not visible in that context). We shouldn't show it in that context.

Resolved here https://github.com/angusmcleod/discourse-translator/commit/ef6a943ebbf468043eef1815a57d7bdacb0c4f20

Reply:
> In fact, I am thinking that it might make sense to not have a separate button for the title but instead just include the title translation when translating a post.

While I can see where you're coming from, I disagree. As mentioned in the [overview video](https://www.loom.com/share/aca110c04d184e9f9666748f54fb8b40), there are different considerations with respect to titles, particularly the topic list that mean it wouldn't make sense to bundle them.